lib/auth/testauthority/testauthority.go

```diff
@@ -29,14 +28,22 @@ import (
 	"github.com/gravitational/teleport/lib/wrappers"
 
 	"github.com/gravitational/trace"
+	"github.com/jonboulle/clockwork"
 	"golang.org/x/crypto/ssh"
 )
 
 type Keygen struct {
+	clock clockwork.Clock
 }
 
+// New creates a new key generator with defaults
 func New() *Keygen {
-	return &Keygen{}
+	return &Keygen{clock: clockwork.NewRealClock()}
+}
+
+// NewWithConfig creates a new key generator with the specified configuration
+func NewWithConfig(clock clockwork.Clock) *Keygen {
```

Comment:
Either rename this to `NewWithClock` or create a `KeygenConfig` struct to wrap the clock.